css/style.css

```diff
@@ -65,12 +75,44 @@ p {
   to {
     transform: rotate(360deg); } }
 
+.background {
+  padding: 0;
+  background: #6070ff no-repeat center;
+  background-size: cover;
+  max-width: 57.5rem;
+  max-height: 88.2rem;
+  height: 100%;
+  width: 100%;
+  backdrop-filter: blur(5px);
+  z-index: 1000;
+  opacity: 0.75;
+  position: absolute;
+  transition: transform 0.8s cubic-bezier(0.83, 0, 0.17, 1); }
+  @media (max-width: 375px) {
```

Comment:
- [ OPTIONAL] Great job so far! 👏🏻  however according to the project requirement, on the desktop size, the hamburger menu button, the closed button, and the menu page must not be visible. Kindly correct this. You can use your @media (min-width: 768px) to display none of your navigation menu.
- I am marking this change as optional because it was missed by the previous reviewer but I strongly suggest considering them.

![000000000](https://user-images.githubusercontent.com/79058364/167899891-afb291c2-3878-4e7b-a251-0ad55df9f29d.png)